ConvenientChests/CraftFromChests/MenuListener.cs

```diff
@@ -38,32 +44,25 @@ public class MenuListener {
 
             switch (e.OldMenu) {
                 case GameMenu _:
-                    GameMenuClosed?.Invoke(sender, e);
                     UnregisterTabEvent();
                     break;
-
-                case CraftingPage _:
-                    if (e.NewMenu is CraftingPage)
-                        break;
-
-                    CraftingMenuClosed?.Invoke(sender, e);
-                    break;
             }
 
             switch (e.NewMenu) {
                 case GameMenu _:
-                    GameMenuShown?.Invoke(sender, e);
-                    this.Events.Display.RenderedActiveMenu += OnRenderedActiveMenu;
+                    Events.Display.RenderedActiveMenu += OnRenderedActiveMenu;
                     break;
 
-                case CraftingPage _:
                 case object m when m.GetType().ToString() == "CookingSkill.NewCraftingPage":
```

Comment:
Probably unnecessary now with the `p.cooking` field, but I couldn't remember if this was from the base game or some other cooking mod